tests/cupyx_tests/scipy_tests/sparse_tests/test_construct.py

```diff
@@ -420,3 +421,38 @@ def test_kron(self, xp, sp):
 
     # TODO(leofang): check oversize inputs as in scipy/scipy#11879 after
     # #3513 is fixed
+
+
+_arrs_kronsum = [
+        [[0]],
+        [[-1]],
+        [[4]],
+        [[10]],
+        [[1, 2], [3, 4]],
+        [[0, 2], [5, 0]],
+        [[0, 2, -6], [8, 0, 14], [0, 3, 0]],
+        [[5, 4, 4], [1, 0, 0], [6, 0, 8]]]
```

Comment:
```suggestion
        [[5, 4, 4], [1, 0, 0], [6, 0, 8]]
        ]
```
If doing this will fix the error?

Reply:
It looks to be an indentation issue, not an issue with the closing bracket.

Reply:
That's right. Unfortunately autopep8 seems to have a stricter requirement on the multi line indentation that flake8 does. Just apply the suggestion given by autopep8 and it should be happy.